includes/Story_Renderer/HTML.php

```diff
@@ -51,6 +51,40 @@ class HTML {
 	 */
 	protected $document;
 
+	/**
+	 * AMP requires the HTML markup to be encoded in UTF-8.
+	 *
+	 * @var string
+	 */
+	const AMP_ENCODING = 'utf-8';
+
+	/**
+	 * Encoding identifier to use for an unknown encoding.
+	 *
+	 * "auto" is recognized by mb_convert_encoding() as a special value.
+	 *
+	 * @var string
+	 */
+	const UNKNOWN_ENCODING = 'auto';
```

Comment:
What is this used for?

Reply:
Ah, I forgot to add it, but this would be the default of `$encoding` further down.

Reply:
Either remove it or use it.

Reply:
As I said, I forgot 😉 So yes, I'll add it.